scripts/config

```diff
@@ -29,21 +32,21 @@ app=$YNH_APP_INSTANCE_NAME
 show_config() {
     # here you are supposed to read some config file/database/other then print the values
     # echo "YNH_CONFIG_${PANEL_ID}_${SECTION_ID}_${OPTION_ID}=value"
-    echo "YNH_CONFIG_SYNAPSE_CONFIG_SERVER_CONFIG_SERVER_STATISTICS=$(ynh_app_setting_get --app $app --key report_stats)"
+    echo "YNH_CONFIG_SYNAPSE_CONFIG_SERVER_CONFIG_SERVER_STATISTICS=$report_stats" >> $YNH_STDRETURN
```

Comment:
You can replace all `echo` by:

```suggestion
    ynh_return "YNH_CONFIG_SYNAPSE_CONFIG_SERVER_CONFIG_SERVER_STATISTICS=$report_stats"
```

https://github.com/YunoHost/yunohost/blob/b74c7c888ce01d516738234b66bf93b1e2a750c2/data/helpers.d/logging#L307-L309

But you need `requirements 3.6` in manifest